scripts/prepare_doc_publication.py

```diff
@@ -72,6 +73,7 @@ def parse_arguments():
     'getting_started_windows_bazel.md': 'GetStarted/WindowsBazel',
     'getting_started_windows_cmake.md': 'GetStarted/WindowsCMake',
     'getting_started_windows_vulkan.md': 'GetStarted/WindowsVulkan',
+    'developer_overview.md': 'DeveloperOverview',
```

Comment:
We probably also want to assign an `navi_order` to this doc in the following directory.